src/pages/filter/ui/attribute-header.tsx

```diff
@@ -0,0 +1,38 @@
+import { ReactElement } from 'react'
```

Comment:
I think it's time to group items in 'filter' folder. It's too many files in here.
Let's rename 'ui' to 'components' and create groups there. Folders for attribute, header, modals et cetera

Reply:
Next stage of refactoring... I tried don't touch any ui components, but some still affected.